src/util/address.rs

```diff
@@ -468,6 +468,15 @@ impl Payload {
             program: output_key.as_inner().serialize().to_vec(),
         }
     }
+
+    /// Returns a byte slice of the payload
```

Comment:
The documentation should be more detailed. I don't think a newbie would be able to understand what it does without reading the code.

Reply:
This looks to be pulled verbatim from existing code that was added in https://github.com/rust-bitcoin/rust-bitcoin/pull/839